config/base.js

```diff
@@ -59,7 +58,7 @@ const SCHEMA_CONFIG = Joi.object()
 module.exports = {
     jobs: SCHEMA_JOBS,
     shared: SCHEMA_SHARED,
-    prChain: SCHEMA_PR_CHAIN,
+    prChain: SCHEMA_PR_CHAIN, // This is still used by pipeline schema.
```

Comment:
Is this  still needed?

Reply:
Yes, for the current models and UI (via pipeline API).
Please refer to the following code.
- https://github.com/screwdriver-cd/models/pull/344/files#diff-679e90b5b15ab1b73ddb7ed2377e7413R637

Reply:
I may be mistaken, but I think the definition of pipeline model is [here](https://github.com/screwdriver-cd/data-schema/blob/master/models/pipeline.js#L67), doesn't it?

Reply:
Yes, the definition of pipeline model refers to this `Base.prChain` here.

Reply:
Oh.. I overlooked :bow: